src/bindings/python/src/openvino/preprocess/torchvision/torchvision_preprocessing.py

```diff
@@ -0,0 +1,325 @@
+class _(TransformConverterBase):
+    def convert(self, input_idx: int, ppp: PrePostProcessor, transform: Callable, meta: Dict) -> None:
+        input_shape = meta["input_shape"]
+        source_size = meta["image_dimensions"]
+        target_size = _setup_size(transform.size, "Incorrect size type for CenterCrop operation")
+
+        if target_size[0] > source_size[0] or target_size[1] > source_size[1]:
+            ValueError(f"CenterCrop size={target_size} is greater than source_size={source_size}")
+
+        bottom_left = []
+        bottom_left.append(int((source_size[0] - target_size[0]) / 2))
+        bottom_left.append(int((source_size[1] - target_size[1]) / 2))
+
+        top_right = []
+        top_right.append(min(bottom_left[0] + target_size[0], source_size[0] - 1))
+        top_right.append(min(bottom_left[1] + target_size[1], source_size[1] - 1))
+
+        bottom_left = [0] * len(input_shape[:-2]) + bottom_left if meta["layout"] == Layout("NCHW") else [0] + bottom_left + [0]  # noqa ECE001
+        top_right = input_shape[:-2] + top_right if meta["layout"] == Layout("NCHW") else input_shape[:1] + top_right + input_shape[-1:]
+
+        ppp.input(input_idx).preprocess().crop(bottom_left, top_right)
+        meta["image_dimensions"] = (target_size[-2], target_size[-1])
+
+
+@TransformConverterFactory.register(transforms.Resize)
+class _(TransformConverterBase):
+    def convert(self, input_idx: int, ppp: PrePostProcessor, transform: Callable, meta: Dict) -> None:
+        resize_mode_map = {
+            InterpolationMode.BILINEAR: ResizeAlgorithm.RESIZE_BILINEAR_PILLOW,
+            InterpolationMode.BICUBIC: ResizeAlgorithm.RESIZE_BICUBIC_PILLOW,
+            InterpolationMode.NEAREST: ResizeAlgorithm.RESIZE_NEAREST,
+        }
+        if transform.max_size:
+            raise ValueError("Resize with max_size if not supported")
+
+        h, w = _setup_size(transform.size, "Incorrect size type for Resize operation")
+
+        ppp.input(input_idx).tensor().set_layout(Layout("NCHW"))
+
+        input_shape = meta["input_shape"]
+
+        input_shape[meta["layout"].get_index_by_name("H")] = -1
+        input_shape[meta["layout"].get_index_by_name("W")] = -1
+
+        ppp.input(input_idx).tensor().set_shape(input_shape)
+        ppp.input(input_idx).preprocess().resize(resize_mode_map[transform.interpolation], h, w)
+        meta["input_shape"] = input_shape
+        meta["image_dimensions"] = (h, w)
+
+
+def _from_torchvision(model: ov.Model, transform: Callable, input_example: Any, input_name: Union[str, None] = None) -> ov.Model:
+
+    if input_name is not None:
+        input_idx = next((i for i, p in enumerate(model.get_parameters()) if p.get_friendly_name() == input_name), None)
+    else:
+        if len(model.get_parameters()) == 1:
+            input_idx = 0
+        else:
+            raise ValueError("Model contains multiple inputs. Please specify the name of the" "input to which prepocessing is added.")
```

Comment:
The message here will be like this: Model contains multiple inputs. Please specify the name of `theinput` to which prepocessing is added.

Reply:
Fixed.